commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -162,6 +229,35 @@ public static CommerceOrder addCheckoutDetailsToUserOrder(
 		return CommerceOrderLocalServiceUtil.updateCommerceOrder(commerceOrder);
 	}
 
+	public static CommerceCatalog addCommerceCatalog(
+			long groupId, String commerceCurrencyCode)
+		throws Exception {
+
+		return CommerceCatalogServiceUtil.addCommerceCatalog(
+			RandomTestUtil.randomString(), commerceCurrencyCode, "en_US", null,
```

Comment:
Might be worth using a constant for `"en_US"` rather than a literal string, maybe something like `LocaleUtil.toLanguageId(Locale.US)`. Not sure if this is worth it though -- just thought I'd mention the possibility.